### ISSUE TITLE
Build warnings using CocoaPods

### ISSUE THREAD

**Issue by drdaz** [Raizlabs/RZBluetooth#88]
I recently upgraded my xcodeproj gem, which has made building projects using CocoaPods much more... sensitive. I now (and perhaps previously) have the following warnings from RZBluetooth.

Most of it can be fixed by updating the deployment target of RZBluetooth to 7.1 I think.

![screen shot 2018-05-17 at 17 44 50](https://user-images.githubusercontent.com/845731/40188481-53691bf8-59fa-11e8-840a-19987966eb07.png)

**Comment by drdaz** [comment]
Alright that was my bad; those warnings were from version 1.0.2.

But the story is similar with the current release. Deploy target needs to be iOS 10 or above now.

**Comment by drdaz** [comment]
... and my project needs to support iOS 9. Egads.

**Comment by drdaz** [comment]
#89

**Comment by KingOfBrian** [comment]
I don't see anything forcing it to be 10.0 and above. What are the warnings for the current release you're seeing? I'd rather not bump to 10+ unless there's a reason.

**Comment by drdaz** [comment]
![screen shot 2018-05-18 at 02 49 32](https://user-images.githubusercontent.com/845731/40210620-2d4c930e-5a46-11e8-9d3f-55097d07751b.png)

**Comment by cpatterson-lilly** [comment]
Ah, yes, we see this build warning as well. It is an incorrect warning, because RZBluetooth has backward-compatibility with the previous `CBCentralManagerState` and `CBPeripheralManagerState` enums built-in. I assume Xcode just flags any occurrence of the symbol as a warning.

We silenced this warning in our builds using this code in our Podfile:
```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == "RZBluetooth.common" || target.name == "RZBluetooth.common-Test" then
        target.build_configurations.each do |config|
            # Silence erroneous RZBluetooth build warnings about `CBManagerState` partial availability
            config.build_settings['CLANG_WARN_UNGUARDED_AVAILABILITY'] = 'NO'
        end
    end
  end
end
```

Perhaps this build setting could be built-in to the RZBluetooth project?

**Comment by KingOfBrian** [comment]
Should be addressed in #95
